atlasdb-commons/src/main/java/com/palantir/common/base/BatchingVisitableView.java

```diff
@@ -136,13 +143,23 @@ public void forEach(final Visitor<T> visitor) {
     }
 
     /**
-     * @return the first element or null if the visitable is empty
+     * Gets the first element in the visitable.
+     *
+     * @return the first element or <code>null</code> if the visitable is empty
      */
     @Nullable
     public T getFirst() {
         return BatchingVisitables.getFirst(delegate());
     }
 
+    /**
+     * Gets the first element in the visitable.
+     *
+     * If a default value of <code>null</code> is wanted, then consider
+     * calling {@link BatchingVisitableView#getFirst()} instead.
+     *
+     * @return the first element or <code>null</code> if the visitable is empty
```

Comment:
@return the first element or <code> defaultElement </code> if the visitable is empty